packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Register an App in Okta
 ---
+
+> **Note:** If you only need to authenticate users into Okta and not have them redirected to a particular OIDC client the IdP configuration would be complete at this point and [routing rules](https://help.okta.com/en/prod/Content/Topics/Security/Identity_Provider_Discovery.htm) have to be set to redirect users from the Okta login page to the correct provider.
```

Comment:
@IanHakes-okta 
If you need Okta only to authenticate users and not to redirect them to a particular OpenID Connect client, then the Identity Provider (IdP) configuration is complete. Add routing rules to redirect users from the Okta Sign-In Page to the IdP.

Reply:
@yuliu-okta